docs/nidigital/enums.rst

```diff
@@ -77,6 +90,79 @@ DigitalState
 
 
 
+DriveEdgeSetFormat
+------------------
+
+.. py:class:: DriveEdgeSetFormat
+
+    .. py:attribute:: DriveEdgeSetFormat.NR
+
+
+
+    .. py:attribute:: DriveEdgeSetFormat.RL
+
+
+
+    .. py:attribute:: DriveEdgeSetFormat.RH
+
+
+
+    .. py:attribute:: DriveEdgeSetFormat.SBC
+
+
+
+HistoryRAMCyclesToAcquire
+-------------------------
+
+.. py:class:: HistoryRAMCyclesToAcquire
+
+    .. py:attribute:: HistoryRAMCyclesToAcquire.FAILED
+
+
+
+    .. py:attribute:: HistoryRAMCyclesToAcquire.ALL
+
+
+
+HistoryRAMTriggerType
+---------------------
+
+.. py:class:: HistoryRAMTriggerType
+
+    .. py:attribute:: HistoryRAMTriggerType.FIRST_FAILURE
+
+
+
+    .. py:attribute:: HistoryRAMTriggerType.CYCLE_NUMBER
+
+
+
+    .. py:attribute:: HistoryRAMTriggerType.PATTERN_LABEL
+
+
+
+PPMUCurrentLimitBehavior
+------------------------
+
+.. py:class:: PPMUCurrentLimitBehavior
+
+    .. py:attribute:: PPMUCurrentLimitBehavior.REGULATE
```

Comment:
Is this correct, an enum of 1 value?
If so, we should consider (whenever we go over defaults) to make this value the default if it makes sense to and other ADEs do so (in nimi-python as a general rule we use default values less often than LabVIEW APIs)

Reply:
Yes, there is only one value for this enum. It is not used in any methods, it's only used in one property. We could make this property private since it's useless but it would be easy to forget to make it public when another value is added; better leave it public.